src/Bootstrap/BootstrapAbstract.php

```diff
@@ -152,6 +195,9 @@ protected function initEnvironment()
     {
         error_reporting(-1);
         date_default_timezone_set('UTC');
+        // TODO: Move these to the docker images [emul]
+        ini_set('xdebug.overload_var_dump', 0);
```

Comment:
Ok, check the next one.  this way it will be turned off only on DEV and for CLI.
What do you think?